### ISSUE TITLE
Moving backward in PHN enrollment loses county contract

### ISSUE THREAD

**Issue by jasonaowen** [SolutionGuidance/psm#734]
tl;dr: hitting "previous" traps you on a page and forgets the previously uploaded attachment

To reproduce,

1. start an enrollment application for a Public Health Nursing Organization
2. upload a county contract on the Facility Credentials page in the "PHN Agency must be a county or have a contract with a county" section
3. move forward in the application by filling out the rest of the page and clicking "Next"
4. click "Previous" to return to the Facility Credentials page
5. click "Next" to move on from this previously-completed page

Expected behavior:
The previously entered data is saved and accepted

Actual behavior:
The contract is lost, the error "Copy of contract with county is required for this provider type." is shown, and the field is highlighted as being in error.

![contract-missing](https://user-images.githubusercontent.com/1494855/37855466-ccd3d768-2ec5-11e8-8820-066b7ad7ed98.png)

Related to issue #158.
